test/python/test_legacy_check_asset_in.py

```diff
@@ -53,7 +53,7 @@ class TestCheckAddressAndDisplay:
 
     @pytest.mark.parametrize("subcommand", [SubCommand.SELL, SubCommand.FUND])
     def test_check_asset_in_legacy_ok(self, backend, exchange_navigation_helper, subcommand):
-        suffix = "_" + str(subcommand).split('.')[1].split('_')[0].lower()
+        suffix = "_" + subcommand.name.split('_')[0].lower()
```

Comment:
Might be worth adding a method to SubCommand to return the operation name

Reply:
yes agreed, in another PR maybe ?

Reply:
Done